cmd/cert-checker/main.go

```diff
@@ -170,10 +175,11 @@ func (c *certChecker) checkCert(cert core.Certificate) (problems []string) {
 		if parsedCert.IsCA {
 			problems = append(problems, "Certificate can sign other certificates")
 		}
-		// Check the cert has the correct validity period +/- an hour
-		if parsedCert.NotAfter.Sub(cert.Issued) > checkPeriod+time.Hour {
+		// Check the cert has the correct validity period
+		expiryPeriod := parsedCert.NotAfter.Sub(parsedCert.NotBefore)
+		if expiryPeriod > checkPeriod {
 			problems = append(problems, "Certificate has a validity period longer than 90 days")
```

Comment:
This and the other append's string ought to use the checkPeriod (or something that created it) to make this text so that when we change the checkPeriod, we change this as well.

It could do so at boot.